### DISCUSSION TITLE
LaserGRBL won't start up after install

### DISCUSSION THREAD

**Discussion by klismont** [arkypita/LaserGRBL#1225] [Q&A]
Hello,

I'm a first time user of LaserGRBL.
After installing LaserGRBL (latest v3.8.6), the software won't start up.
I've tried rebooting my computer, tried installing earlier releases, no luck.
My computer:

![image](https://user-images.githubusercontent.com/61911640/103150725-ea723b80-4776-11eb-8cff-5d2d537e3b4e.png)

In another post, the suggestion was made to delete the content of the LaserGRBL folder in C:\Users\*USER*\AppData\Roaming, but there is no such folder strangely enough...

![image](https://user-images.githubusercontent.com/61911640/103150696-bb5bca00-4776-11eb-987b-f900fb701b3d.png)

I hope you have any other suggestions.

Thanks in advance,
Kevin

**Comment by arkypita** [comment]
Hi! The folder C:\Users*USER*\AppData\Roaming\LaserGRBL contains lasergrbl user options and settings, and only exist after the first run.

Verify that your antivirus is not blocking LaserGRBL from starting up.
Also LaserGRBL need Microsoft .NET framework 3.5 or earlier installed on PC. Verify you have it.

Please use "[discussions](https://github.com/arkypita/LaserGRBL/discussions)" and not issue page. (moved)
